esp/useful_scripts/server_setup/new_site.sh

```diff
@@ -365,28 +371,13 @@ then
     read THROWAWAY
 fi
 
-# Mailman setup
-# To reset: remove list manually
-if [[ "$MODE_MAILMAN" || "$MODE_ALL" ]]
-then
-    newlist -q $SITENAME-websupport $ADMINEMAIL $MAILMAN_LIST_PASSWORD
-    add_members -r - granite-websupport <<EOF
-$ADMINEMAIL
-EOF
-    echo "Created Mailman list $SITENAME-websupport with initial member $ADMINEMAIL."
-    
-    echo "Support list has been created.  Please check them by looking over the"
-    echo -n "output above, then press enter to continue or Ctrl-C to quit."
-    read THROWAWAY
-fi
-
 # Apache setup
 # To reset: remove appropriate section from Apache config
 if [[ "$MODE_APACHE" || "$MODE_ALL" ]]
 then
     cat >>$APACHE_CONF_FILE <<EOF
 #   $INSTITUTION $GROUPNAME (automatically generated)
-WSGIDaemonProcess $SITENAME processes=1 threads=1 maximum-requests=1000
+WSGIDaemonProcess $SITENAME processes=2 threads=1 maximum-requests=1000
```

Comment:
There's probably a way to make sure the processes get killed if they're idle for too long.  I'm not entirely sure whether the `inactivity-timeout` option to `mod_wsgi`'s `WSGIDaemonProcess` is exactly what we want -- it's unclear to me if that is for the whole process group or for individual processes -- but it should work.  We should probably have an Apache global `MaxClients` setting anyway, so AIUI this would only be necessary to prevent one site from eating all the processes and choking another.